coss/home/models.py

```diff
@@ -17,22 +17,42 @@ class Meta:
 
 @register_snippet
 class AboutSnippet(models.Model):
+    title = models.CharField(default='', max_length=40)
     header_text = RichTextField(blank=True)
     body_title = models.CharField(blank=True, default='', max_length=40)
     body_text = RichTextField(blank=True)
     info_title = models.CharField(blank=True, default='', max_length=40)
     info_text = RichTextField(blank=True)
+    bottom_cta_title = models.CharField('Title', blank=True, default='', max_length=40)
+    bottom_cta_text = models.CharField('Text', blank=True, default='', max_length=40)
+    bottom_cta1_text = models.CharField('Text', blank=True, default='', max_length=20)
+    bottom_cta1_link = models.URLField('Link', blank=True, default='')
+    bottom_cta2_text = models.CharField('Text', blank=True, default='', max_length=20)
+    bottom_cta2_link = models.URLField('Link', blank=True, default='')
```

Comment:
Where there is such repetition, it's probably a good place to optimize things a bit. Ideally we can have a small block/snippet with its own template that can be added an arbitrary number of times to multiple places. In our case, we can have it here as a ForeignKey but also as a standalone version for other templates.

That said, I am OK with leaving this as it is for now and we can polish it a bit later. I opened the issues #325 and #326 to track this effort.